### ISSUE TITLE
中英文文档不匹配和测试案例与版本不匹配的问题

### ISSUE THREAD

**Issue by jhcgt4869** [PaddlePaddle/Paddle#50424]
### 文档链接&描述 Document Links & Description

paddle.nn.CrossEntropyLoss的中文文档信息确实的情况。
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/CrossEntropyLoss_cn.html#crossentropyloss
与
https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/CrossEntropyLoss_en.html#crossentropyloss
还有
https://github.com/PaddlePaddle/Paddle/blob/release/2.4/python/paddle/nn/layer/loss.py#L141

分别是中文的文档，英文文档与代码注释，三方无法匹配，特别是中文和英文的差距极大。
同时还有测试代码里面是结果与当前情况不匹配。
![image](https://user-images.githubusercontent.com/61859193/218088172-34a6ed49-8c96-4103-81b9-280e94299aeb.png)
![image](https://user-images.githubusercontent.com/61859193/218088210-4abeb4b3-eaf1-4420-ae7b-9d682c113215.png)

同时看了该接口的上一级的接口 `cross_entropy`
其中英文文档信息也不完全匹配。

### 请提出你的建议 Please give your suggestion

建议可以进行修改和完善，对于小的问题也可以及时改正。
理解团队的辛苦也修改的工作量大，存在许多历史遗留问题，但是还是希望可以进行改进

**Comment by lyuwenyu** [comment]
收到 多谢反馈

**Comment by luotao1** [comment]
@jhcgt4869 欢迎提个PR来修复下！
